people_tool/static/people_tool/js/controllers/SearchController.js

```diff
@@ -98,9 +98,23 @@
             // can still display a clickable anchor link
             var lastName = (full.name_last.trim().length > 0) ?
                 full.name_last : '(none)';
-            return '<a href="' + '#/people/' + full.univ_id + '/courses/">'
+            return '<a href="' + '#/people/' + full.univ_id + '/courses/" ' +
+                'ng-click="setSelectedPersonInfo(\''+full.name_last+'\',\''+
+                full.name_first+'\','+full.univ_id+',\''+full.email_address+'\''+',\''+full.role_type_cd+'\')">'
```

Comment:
Or perhaps instead of an external <a> link, we can simply use `ng-click` to call an angular controller function which sets the person in the service and changes `$location` to load the `#/people/univ_id/courses/` route.